### PR TITLE
[ARM32/Linux] Fix definition and CMake for cross-architecture build

### PR DESCRIPTION
**Pull request by hseok-oh**
- Fix ABI flag based on target
- Exclude src/corefx for cross-architecture component

### CODE REVIEW

**Comment by hseok-oh** [comment]
related: https://github.com/dotnet/coreclr/pull/8866#issuecomment-276843728

**Comment by hseok-oh** [comment]
@janvorli PTAL

File: src/corefx/System.Globalization.Native/CMakeLists.txt [hunk 1]
```diff
@@ -115,5 +115,5 @@ verify_dependencies(
 
 # add the install targets
 install_clr(System.Globalization.Native)
-install(TARGETS System.Globalization.Native_Static DESTINATION .)
+install_clr(System.Globalization.Native_Static)
```
Comment (janvorli) [inline src/corefx/System.Globalization.Native/CMakeLists.txt]: This change doesn't seem right. The install_clr strips debug symbols and installs PGO related file. None of this applies to static libraries. Moreover, neither the shared nor the static version of the System.Globalization.Native is needed for the cross components, so I think the whole src/corefx should be excluded for cross components build.

Comment (hseok-oh) [inline src/corefx/System.Globalization.Native/CMakeLists.txt]: @janvorli I agree. I'll fix to exclude src/corefx for cross component.

**Comment by hseok-oh** [comment]
`install_clr` and `_install` exclude installation for cross component build, so I change `install_clr` to `_install` for static `System.Globalization.Native` and keep `install_clr` for shared version.

**Comment by janvorli** [comment]
@hseok-oh but why do we build the System.Globalization.Native.so at all in the cross components build? I was thinking that you would exclude it from building by excluding `add_subdirectory(src/corefx)` in the root CMakeLists.txt for cross components build. Is there a problem with that?

**Comment by hseok-oh** [comment]
@janvorli You're right. It's better to exclude it in root CMakeLists.txt.

**Comment by hseok-oh** [comment]
@janvorli Is there anything to do more?

**Pull request merged**